docs/GettingStartedDocs/Contributors/ExperimentalSupportRHEL8.md

```diff
@@ -0,0 +1,99 @@
+# Experimental - Getting Started with Open Enclave on RHEL8
+
+## Platform requirements
+
+- RHEL8
+- A system which is SGX1 or SGX1+FLC capable. Please refer to [SGXSupportLevel.md](/docs/GettingStartedDocs/SGXSupportLevel.md) to determine the SGX support level for your target system.
```

Comment:
To confirm: So simulation mode on a non-SGX machine is NOT available on RHEL8, is that correct?

Reply:
Simulation mode is available on RHEL. Updated the documentation.